config/initializers/configuration.rb

```diff
@@ -1,37 +1,97 @@
 Workarea::Configuration.define_fields do
   fieldset 'Affirm', namespaced: true do
     field 'Enabled',
+      id: :enabled,
       type: :boolean,
       description: 'Whether to enable Affirm functionality throughout the site',
       default: true
 
     field 'Public Key',
+      id: :public_key,
       type: :string,
       description: 'Public API key. Found in the Affirm API Keys admin section.',
       allow_blank: true,
       encrypted: false
 
     field 'Private Key',
+      id: :private_key,
       type: :string,
       description: 'Private API key used for server side calls. Found in the Affirm API Keys admin section.',
       allow_blank: true,
       encrypted: true
 
     field 'Use Production Environment',
+      id: :use_production_environment,
       type: :boolean,
       description: 'Uses the production API and Javascript SDK URLS.',
       default: false
 
     field 'Merchant Name',
+      id: :merchant_name,
       type: :string,
       description: 'Name of merchant that display in Affirm checkout.',
       allow_blank: true,
       encrypted: false
 
     field 'Minimum Order Value',
+      id: :minimum_order_value,
       type: :float,
       description: 'Minimum order total required for Affirm. Affirm will not show as a payment option if this threshold is not met. Leave this field blank for no minimum.',
       allow_blank: true,
       encrypted: false
+
+    field 'Modal Checkout',
+      id: :modal_checkout,
+      type: :boolean,
+      description: 'Enable the modal dialog in checkout rather than redirecting offsite.',
+      default: false
+
+    field 'Capturing',
+      id: :capturing,
+      type: :boolean,
+      description: 'Enable capturing through Affirm. The default is authorization only.',
+      default: false
+
+    field 'Show PDP Messaging',
+      id: :show_pdp_messaging,
+      type: :boolean,
+      description: 'Enable Affirm messaging on the product page',
+      default: true
+
+    field 'Show Cart Messaging',
+      id: :show_cart_messaging,
+      type: :boolean,
+      description: 'Enable Affirm messaging on the cart',
+      default: true
+
+    field 'PDP Logo Color',
+      id: :pdp_logo_color,
+      type: :string,
```

Comment:
So they just type any color? Is it hex or name or what? How will the user know what to put?

Reply:
i totally forgot the `:values` for like 4 of these lol

Reply:
also removed the `:default` setting, as [Affirm will provide defaults when these values are not set](https://docs.affirm.com/affirm-developers/docs/html-reference).